app/filtering.js

```diff
@@ -357,7 +352,7 @@ function registerPermissionHandler (session, partition) {
         {text: locale.translation('deny')},
         {text: locale.translation('allow')}
       ],
-      frameOrigin,
+      origin,
```

Comment:
I guess I'm confused about how this should work. The `origin` is the origin of the url that is actually requesting the permission. If we used `mainFrameUrl` here it would appear to the user that the main page is requesting permission when it may actually be requested by a frame with a different origin